test/vm-run

```diff
@@ -25,12 +25,17 @@ import testinfra
 parser = argparse.ArgumentParser(description='Run a test machine')
 parser.add_argument('-v', '--verbose', action='store_true', help='Display verbose details')
 parser.add_argument('-m', '--maintain', action='store_true', help='Changes are permanent')
+parser.add_argument('-r', '--ram', default=None, type=int, help='Memory (in MiB) of the target machine')
```

Comment:
Can we just use the long form here, and for ```--cpus``` below? I say this because '-r' typically means recursive in an argument list. If you really want short letters, lets use capitals for these. ie: ```-M``` and ```-C```